getting_started.rst

```diff
@@ -270,7 +454,7 @@ parameter. As an exercise, try building your timer project with a different conf
 
 .. code-block:: bash
 
-    $ conan install . -s arch=x86
+    $ conan install . --settings arch=x86
```

Comment:
(Look at the text line explaining this command) What if the default was a x86 architecture? I would use something related to the *gcc_x86* profile already mentioned:

```
conan install . --profile gcc_x86 --settings arch=x86_64
```

Reply:
Also, in the line below, use the same param, not the short one: ``--settings arch=****``